YourEasyRent/Services/ITelegramMenu.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Telegram.Bot;
+using Telegram.Bot.Types.ReplyMarkups;
+
+namespace YourEasyRent.Services
+{
+    public interface ITelegramMenu
+    {
+        InlineKeyboardMarkup meinMenu { get; }
+        InlineKeyboardMarkup brandMenu { get; }
+        InlineKeyboardMarkup categoryMenu { get; }
```

Comment:
All of them are menus in a TelegramMenu class, so you can just name them => Main, Category, Brand